WordPressCom-Stats-iOS/StatsEphemory.m

```diff
@@ -13,7 +13,7 @@ @implementation EphemoryContainer
 
 @interface StatsEphemory ()
 
-@property (nonatomic, strong) NSCache *cache;
+@property (nonatomic, strong) NSMutableDictionary *cache;
```

Comment:
Every time the app would background, the NSCache instance would get automatically evicted. I needed something a little more persistent. I'm not serializing this one to disk (yet) but I plan on doing something like that soon.